### PR TITLE
Library branch working

### PR DESCRIPTION
**Pull request by arslanbisharat**
### PROJECT: LIBRARY

> All the instructions are given [**Here**](https://www.theodinproject.com/courses/javascript/lessons/library) are followed in the project
>>  I completed this project alone

### CODE REVIEW

**Comment by bellom** [comment]
> ### PROJECT: LIBRARY
> > All the instructions are given [**Here**](https://www.theodinproject.com/courses/javascript/lessons/library) are followed in the project
> > > I completed this project alone

Nice short details but you need to list the tasks you performed.
Example: 
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/

File: library.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    this.title = title;
+    this.author = author;
+    this.pageNumber = pageNumber;
+    this.read = read;
+  }
+
+  info () {
+    return `${this.title} by ${this.author}, ${this.pageNumber} pages, ${this.read}`;
+  }
+}
```
Comment (bellom) [inline library.js]: wow, App interface really looks nice
but Opps! just one more thing to do 👇 
Let's separate the DOM from application logic, you can create another file for DOM.

image

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623451/

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @arslanbisharat  :clap: 

#### May I know the reason why you're working alone on a collaborative project?  
#### You needs to answer this before moving on, kindly explain, thanks.

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Congratulations :clap: 

Thank you for applying the suggested changes.
You have a really good code now.

However, as you didn't explain why you are working alone as requested by @bellom, the last reviewer, the project can be validated.

Kindly do it please.

Mael FOSSO

**Comment by arslanbisharat** [comment]
@maelfosso @bellom  This was an issue with my repair request. I have already addressed this issue to Microverse team.

**Pull request merged**